storage/backup/backup-cli/src/coordinators/backup.rs

```diff
@@ -353,10 +360,54 @@ impl BackupCompactor {
             state_snapshot_file_compact_factor,
             transaction_file_compact_factor,
             concurrent_downloads,
+            remove_compacted_files_after,
         }
     }
 
-    pub async fn run(&self) -> Result<()> {
+    /// Update the existing mapping and return the files to be moved out of metadata folder
+    fn update_compaction_timestamps(
```

Comment:
looks like you are not invoking this as part of backup-continuously? then let's move the backup compactor to a separate file (in a separate PR).

Reply:
yes, I will create a separate PR after this merges.